### ISSUE TITLE
@rollup/plugin-replace does not replace files in collection

### ISSUE THREAD

**Issue by BerkanY** [ionic-team/stencil#3019]
**Stencil version: 6.14.14**

```
 @stencil/core@2.5.2 
```

**I'm submitting a:**

[x] bug report

**Current behavior:**
I use the @rollup/plugin-replace Plugin in the stencil.config.ts to replace the component version with the one from the package.json. It works fine (e.g in dist/esm, dist/esm-es5) except the dist/collection directory. I can still see still the placeholder not the version.

**Expected behavior:**
I thought each file which is included to the build proccess is going to replaced with this plugin, when athe Placeholder is found. 

**Steps to reproduce:**

see code below

**Related code:**
stencil.config.ts
```tsx
export const config: Config = {
  namespace: 'links',
  taskQueue: 'async',
  buildEs5: true,
  rollupPlugins:{ 
    after: [
      replace({
        delimiters: ['<@', '@>'],
        preventAssignment: true,
        values: {
          PACKAGE_VERSION: version
        }
      })
  ]},
  outputTargets: [
    {
      type: 'dist',
      esmLoaderPath: '../loader'
    },
    {
      type: 'docs-readme'
    }
  ],
  testing: {
    browserArgs: ['--no-sandbox', '--disable-setuid-sandbox']
  }
};
```
package.json
```json
...
"main": "./dist/index.cjs.js",
  "module": "./dist/index.js",
  "es2015": "dist/esm/index.mjs",
  "es2017": "dist/esm/index.mjs",
  "types": "dist/types/index.d.ts",
  "collection": "dist/collection/collection-manifest.json",
  "collection:main": "dist/collection/index.js",
  "unpkg": "dist/links/links.js",
  "files": [
    "dist/",
    "loader/"
  ],
  "scripts": {
    "build": "stencil build",
    "build.watch": "stencil build --dev --docs --watch",
    "start": "stencil build --dev --watch",
    "test": "stencil test --spec --e2e --no-sandbox",
    "test.watch": "stencil test --spec --e2e --watchAll",
    "generate": "stencil generate"
  },
...
```

**Comment by splitinfinities** [comment]
Hey @BerkanY - thank you for the report. Can you do me a favor? Try placing the replace plugin within the plugins config, like so: https://github.com/barcamp-events/app/blob/next/stencil.config.ts#L60

I suspect `after` works correctly for your needs. I'm curious if you can confirm that placing it within the plugins config would solve the problem of replacing content in the collections directory. 

If it does solve your problem, do me a big favor and create a Github reproduction repo with this issue, exactly how you have it written within the rollupPlugins.

**Comment by BerkanY** [comment]
@splitinfinities  unfortunately it does not solve the problem. After the suggested adjustments the files in the dist/collection still have the placeholder not the version. All other files have the version after build.

**Comment by rwaskiewicz** [comment]
@BerkanY would it be possible for you to put together a full reproduction repo for us? This would allow me to pull down the reproduction instead of putting one together manually (where I may accidentally miss something that's necessary to reproduce this issue in full)? I appreciate the snippets you have in the GitHub summary there, I just don't want to miss anything 😉

**Comment by rwaskiewicz** [comment]
This issue is being closed due to the lack of a code reproduction. If this is still an issue with the latest version of Stencil, please create a new issue and ensure the template is fully filled out.

Thank you for using Stencil!
